### ISSUE TITLE
Nuget package version 1.2.1 doesn't support .netcore nor .netstandard

### ISSUE THREAD

**Issue by ThomasBeavers** [tjscience/RoboSharp#88]
Version 1.2.0 of the nuget package included support for .netcore and .netstandard, but that has been lost in version 1.2.1.

**Comment by moon6969** [comment]
Still an issue in 1.2.2.

**Comment by RFBomb** [comment]
This code block is disabled in in .Net 5.0

RoboCommand.cs

```
#if NET45
        public async Task<Results.RoboCopyResults> StartAsync(string domain = "", string username = "", string password = "")
        {
            await Start(domain, username, password);
            return GetResults();
        }
#endif
```

The reason _why_  its unavailable is obvious, but whats the reason behind the preprocessor directive?

**Comment by PCAssistSoftware** [comment]
Is this still an issue in latest NuGet release?

**Comment by ThomasBeavers** [comment]
> Is this still an issue in latest NuGet release?

I am now seeing the following supported frameworks as part of the published 1.2.5 version from nuget.
.NETCoreApp 3.1
.NETFramework 4.0
.NETFramework 4.5
.NETStandard 2.0
.NETStandard 2.1

I can confirm that the .NETCoreApp 3.1 version restored and worked for me.

**Comment by PCAssistSoftware** [comment]
Thanks for confirming :)
